.gitignore

```diff
@@ -11,3 +11,7 @@
 *.log
 *.swp
 .byebug_history
+
+# IDEA
+*.idea
+*.iml
```

Comment:
We don't add editor specific rules to the project gitignore. You can add global rules to ignore such files in _all_ repos.